core/ts.core/src/ts/utils/ZipUtils.java

```diff
@@ -172,12 +173,22 @@ public static void extractTar(File file, File destination) throws IOException {
 					break;
 				case TarEntry.LINK:
 					File linkFile = new File(destination, outFilename);
+					// Be sure that parent file exists
+					File linkBaseDir = linkFile.getParentFile();
+					if (!linkBaseDir.exists()) {
+						linkBaseDir.mkdirs();
+					}
 					Path target = new File(linkFile.getParentFile(), entry.getLinkName()).toPath();
```

Comment:
Probably...  but as sample file didn't have a "link" (only "symlinks") I didn't changed that part.  
I'll try to test with new .tar.tz file that includes a link.